tests/docker/php.ini

```diff
@@ -0,0 +1,11 @@
+
+xdebug.mode=debug
```

Comment:
Doesn't look like we need it in this mode. It slows tests down. Was that added for development purposes?